isis/tests/FunctionalTestsJigsaw.cpp

```diff
@@ -808,3 +852,254 @@ TEST_F(ObservationPair, FunctionalTestJigsawErrorTBParamsNoSolve) {
+    DecVelocitySigma=0.0
+    DecAccelerationValue=0.0
+    DecAccelerationSigma=1.0
+  EndGroup
+  Group = "PRIME MERIDIAN"
+    Pm=velocity
+    PmValue=38.32132
+    PmSigma=0.0
+    PmVelocityValue=13.17635815
+    PmVelocitySigma=0.0
+    PmAccelerationValue=0.0
+    PmAccelerationSigma=1.0
+  EndGroup
+  Group = "RADII"
+    RadiiSolveOption=triaxial
+    RadiusAValue=1737400
+    RadiusASigma=0.0
+    RadiusBValue=1737400
+    RadiusBSigma=0.0
+    RadiusCValue=1737400
+    RadiusCSigma=0.0
+    MeanRadiusValue=1737400
+    MeanRadiusSigma=0.0
+  EndGroup
+EndObject
+End)");
+
+  Pvl tbParams; 
+  tbPvlStr >> tbParams; 
+  tbParams.write(tbParamsPath);
+
+  QString outCnetFileName = prefix.path() + "/outTemp.net";
+
+  for(int i = 0; i < cubes.size(); i++) {
+      Pvl *label = cubes[i]->label();
+      // get body rotation
+      PvlObject &br = label->object(4);
+      std::cout << br << std::endl;
+      PvlKeyword ra("PoleRa");
+      ra+= "269.9949";
+      ra+= "0.036";
+      ra+= "0.0"; 
+
+      PvlKeyword dec("PoleDec");
+      dec += "66.5392";
+      dec += "0.0130";
+      dec += "0.0"; 
+
+      PvlKeyword pm("PrimeMeridian");
+      pm += "38.3213";
+      pm += "13.17635815";
+      pm += "1.4E-12";
+
+      PvlKeyword &ft = br.findKeyword("FrameTypeCode");
+      ft.setValue("2");
+
+      br.addKeyword(ra);
+      br.addKeyword(dec);
+      br.addKeyword(pm);
+      cubes[i]->close();
+      delete cubes[i];
+      cubes[i] = nullptr;
+  }
+
+  // just use isdPath for a valid PVL file without the wanted groups
+  QVector<QString> args = {"fromlist="+cubeListFile, "cnet="+controlNetPath, "onet="+outCnetFileName, 
+                          "Solvetargetbody=yes", "Errorpropagation=yes",  "Camsolve=angles", "twist=off", "camera_angles_sigma=2.0", "bundleout_txt=yes", 
+                          "imagescsv=no", "output_csv=no", "residuals_csv=no", "file_prefix="+prefix.path()+"/", "tbparameters="+tbParamsPath};
+
+  UserInterface options(APP_XML, args);
+  
+  Pvl log; 
+  
+  try {
+    jigsaw(options, &log);
+  }
+  catch (IException &e) {
+    FAIL() << "Failed to bundle: " << e.what() << std::endl;
+  } 
+
+  QFile bo(prefix.path() + "/bundleout.txt");
+  QString contents; 
+  if (bo.open(QIODevice::ReadOnly)) {
+    contents = bo.read(bo.size()); 
+  }
+  else { 
+    FAIL() << "Failed to open bundleout.txt" << std::endl;
+  }
+
+  QStringList lines = contents.split("\n");
+
+  EXPECT_THAT(lines[75].toStdString(), HasSubstr("RADII: TRIXIAL"));
+  EXPECT_PRED_FORMAT2(AssertQStringsEqual, lines[76].trimmed(), "");
+  
+  // check residual stats 
+  EXPECT_THAT(lines[132].toStdString(), HasSubstr("minimum: -178.872"));
+  EXPECT_THAT(lines[133].toStdString(), HasSubstr("Quartile 1:   -9.638"));
+  EXPECT_THAT(lines[134].toStdString(), HasSubstr("Median:   +1.377"));
+  EXPECT_THAT(lines[135].toStdString(), HasSubstr("Quartile 3:   +9.449"));
+  EXPECT_THAT(lines[136].toStdString(), HasSubstr("maximum: +175.731"));
```

Comment:
I would expect these two tests to check the values put in the bundlout.txt file by the BundleTargetBodty object, see https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/control/objs/BundleUtilities/BundleTargetBody.cpp#L958 these are the actual values we are solving for.